starter/source/elements/initia/hm_read_inistate_d00.F

```diff
@@ -185,6 +186,7 @@ SUBROUTINE HM_READ_INISTATE_D00 (
       ALLOCATE (ITRIQ(NUMELQ))
       ALLOCATE (INDEXQ(2*NUMELQ))
       ALLOCATE (KSYSUSRQ(2*NUMELQ))
+      ALLOCATE (IES2IPARG(NUMELS))
```

Comment:
Please use MY_ALLOCATE or check the status of the allocation ( cf do & don't s)

Reply:
done